### ISSUE TITLE
About OntoNotes 5.0

### ISSUE THREAD

**Issue by hary00078** [Nealcly/LAN#14]
Hello. 

I have a question about result when I use OntoNotes 5.0 as dataset for NER task.
As you wrote the performance of BiLSTM-LAN, expected accuracy is 88.16%..
But when I implemented model, I got 91.85% at Epoch 1.
Is there any mistake in my command? 
My command is here.

 python main.py --learning_rate 0.01 --lr_decay 0.035 --dropout 0.5                                                                                                              --hidden_dim 400 --lstm_layer 3 --momentum 0.9 --whether_clip_grad True --clip_                                                                                                             grad 5.0 --train_dir 'data/onto.train.txt' --dev_dir 'data/onto.development.txt'                                                                                                              --test_dir 'data/onto.test.txt' --model_dir 'model/' --word_emb_dir 'glove.6B.1                                                                                                             00d.txt'

And this is Summary

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
DATA SUMMARY START:
 I/O:
     Tag          scheme: BIO
     MAX SENTENCE LENGTH: 250
     MAX   WORD   LENGTH: -1
     Number   normalized: False
     Word  alphabet size: 69812
     Char  alphabet size: 119
     Label alphabet size: 38
     Word embedding  dir: glove.6B.100d.txt
     Char embedding  dir: None
     Word embedding size: 100
     Char embedding size: 30
     Norm   word     emb: False
     Norm   char     emb: False
     Train  file directory: data/onto.train.txt
     Dev    file directory: data/onto.development.txt
     Test   file directory: data/onto.test.txt
     Raw    file directory: None
     Dset   file directory: None
     Model  file directory: model/
     Loadmodel   directory: None
     Decode file directory: None
     Train instance number: 2200752
     Dev   instance number: 304684
     Test  instance number: 230111
     Raw   instance number: 0
     FEATURE num: 0
 ++++++++++++++++++++++++++++++++++++++++
 Model Network:
     Model        use_crf: False
     Model word extractor: LSTM
     Model       use_char: True
     Model char extractor: LSTM
     Model char_hidden_dim: 50
 ++++++++++++++++++++++++++++++++++++++++
 Training:
     Optimizer: SGD
     Iteration: 100
     BatchSize: 4
     Average  batch   loss: False
 ++++++++++++++++++++++++++++++++++++++++
 Hyperparameters:
     Hyper              lr: 0.01
     Hyper        lr_decay: 0.035
     Hyper         HP_clip: None
     Hyper        momentum: 0.9
     Hyper              l2: 1e-08
     Hyper      hidden_dim: 400
     Hyper         dropout: 0.5
     Hyper      lstm_layer: 3
     Hyper          bilstm: True
     Hyper             GPU: True
DATA SUMMARY END.
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

This is a sample of dataset. I convert data format to txt file.

Just	O
now	O
we	O
were	O
primarily	O
talking	O
about	O
those	O
produced	O
by	O
counterfeiting	O
 
Thanks you.

**Comment by Nealcly** [comment]
Hi,
Please add --seg True, because NER is a segmentation task. The model should be evaluated using f1.

**Comment by hary00078** [comment]
Thank you for the reply. I'll try it right away.
